pinot-query-runtime/src/test/resources/queries/TableExpressions.json

```diff
@@ -21,6 +21,9 @@
       { "sql": "SELECT * FROM {tbl} WHERE intCol IN (196883, 42)" },
       { "sql": "SELECT * FROM {tbl} WHERE intCol NOT IN (196883, 42) AND strCol IN ('alice')" },
       { "sql": "SELECT * FROM {tbl} WHERE strCol IN (SELECT strCol FROM {tbl} WHERE intCol > 100)" },
+      { "sql": "SELECT * FROM {tbl} WHERE strCol IN (SELECT strCol FROM {tbl} WHERE intCol < 100)" },
+      { "sql": "SELECT * FROM {tbl} WHERE strCol NOT IN (SELECT strCol FROM {tbl} WHERE intCol > 100)" },
```

Comment:
can we test `NOT IN` with various different types?

Reply:
will test that, but actually `NOT IN` failed on a different issue than the leaf stage, so will break out that in separate PR